### PR TITLE
fix(slider): add pointer move handling; clear old HammerJS pan event.

### LINKED ISSUE #14036: Slider doesn't slide
## Description  
The slider component no longer slides

 * igniteui-angular version: master branch
 * browser: all

## Steps to reproduce  

1. Open the slider sample in the demo app
2. Try to move the slider handle using the mouse

## Result  
The slider doesn't react and the handle(s) is stuck.

## Expected result  
The slider handle should follow the mouse.

### PR DESCRIPTION
**Pull request by skrustev**
Closes #14036

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [x] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [x] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Review by dkamburov — APPROVED** [review]

File: projects/igniteui-angular/src/lib/slider/slider.component.ts [hunk 1]
```diff
@@ -1054,6 +1057,11 @@ export class IgxSliderComponent implements
                 takeUntil(this._destroyer$)).subscribe(() => this._ngZone.run(() => {
                     this.stepDistance = this.calculateStepDistance();
                 }));
+            fromEvent(this._el.nativeElement, 'pointermove').pipe(
+                throttle(() => interval(0, animationFrameScheduler)),
+                takeUntil(this._destroyer$)).subscribe(($event: PointerEvent) => this._ngZone.run(() => {
```
Comment (damyanpetev) [inline projects/igniteui-angular/src/lib/slider/slider.component.ts:1062]: Not sure why `zone.run` would be required, the event is not subscribed is `runOutsideAngular` or anything.

File: projects/igniteui-angular/src/lib/slider/slider.component.ts [hunk 2]
```diff
@@ -804,13 +805,22 @@ export class IgxSliderComponent implements
...
         activeThumb.nativeElement.setPointerCapture($event.pointerId);
         this.showSliderIndicators();
 
         $event.preventDefault();
     }
 
+    /**
+     * @hidden
+     */
+    public onPointerMove($event: PointerEvent) {
```
Comment (damyanpetev) [inline projects/igniteui-angular/src/lib/slider/slider.component.ts:819]: This shouldn't be needed for much else, so either `private` or `protected` if used in template
```suggestion
    private onPointerMove($event: PointerEvent) {
```

**Review by damyanpetev — CHANGES_REQUESTED** [review]

File: projects/igniteui-angular/src/lib/slider/slider.component.ts (hunk 1 above)
Comment (skrustev) [reply projects/igniteui-angular/src/lib/slider/slider.component.ts:1062]: A few lines above you can see that it is ran outside angular actually.

Comment (damyanpetev) [reply projects/igniteui-angular/src/lib/slider/slider.component.ts:1062]: Ah lol didn't expand the diff up enough to see it

File: projects/igniteui-angular/src/lib/slider/slider.component.spec.ts [hunk 3]
```diff
@@ -330,61 +332,6 @@ describe('IgxSlider', () => {
             fixture.detectChanges();
             expect(Math.round(slider.value as number)).toBe(60);
         });
-
-        it('should not set value if value is nullish but not zero', () => {
```
Comment (damyanpetev) [inline projects/igniteui-angular/src/lib/slider/slider.component.spec.ts:334]: Um these tests intentionally removed? Seem unrelated.

**Review by damyanpetev — APPROVED** [review]

**Pull request merged**